client/landing/site.landing/coffee/teams/findteamview.coffee

```diff
@@ -55,21 +57,32 @@ module.exports = class FindTeamView extends kd.TabPaneView
     utils.findTeam email,
       error       : (xhr) =>
         { responseText } = xhr
-        new kd.NotificationView { title : responseText }
+        @showNotification @formatServerError responseText
         @form.button.hideLoader()
       success     : =>
         @form.button.hideLoader()
         @form.reset()
 
-        new kd.NotificationView
-          cssClass : 'recoverConfirmation'
-          title    : 'Check your email'
-          content  : 'We\'ve sent you a list of your teams.'
-          duration : 4500
+        @showNotification 'Check your email', 'We\'ve sent you a list of your teams.'
 
         kd.singletons.router.handleRoute '/'
 
 
+  formatServerError: (err) ->
+
+    return 'You don\'t participate in any team'  if err is EMPTY_TEAM_LIST_ERROR
```

Comment:
also please make sure that the last line of the email is only shown to people who participate more than one teams.

![img](https://d1ro8r1rbfn3jf.cloudfront.net/ms_37620/jCUmpbW6o9it9OJfqpkB4sR0LkHZnZ/Slack%2B2016-09-23%2B21-58-30.png?Expires=1474779596&Signature=TU4Wvz2co1oJH5taYXLFBXDbyLxEaLksLJBSEQEFaPHeP0FNVy~OP5yCYYtOaLbCREWDS4~NsXXqsvdSv4FrD-oRXFR9zwRuIEWav0Z0xKpQsEdHi85OuaQTsqUNTMyw6DQWRAjE60sMfL47j~KikYuhJgLo76uJPsOl3gLluIL0z78VDTjK8epk9705E0SiJ6nkY9KZGdRAEIfIQkFz3U5DW9cb~u8N9ZzdMJZjjQnj6kd6~nRdb8ucgQvsF0uxGe3ZNidAZdfTxl-PUhRNrXaYFMAquKbCpvEyclcC5U6v~HFWvJocCsqPQE7pEhE7Q5j7vohd9qCHVhpNT3BOWw__&Key-Pair-Id=APKAJHEJJBIZWFB73RSA)